helm/hpcc/templates/_helpers.tpl

```diff
@@ -615,3 +615,41 @@ resources:
     memory: "100M"
 {{- end -}}
 
+
+{{/*
+Create placement related settings
+*/}}
+{{- define "hpcc.doPlacement" -}}
+{{- if .placement.nodeSelector }}
```

Comment:
Is this function essential generating everything in a placement, except 'name' and 'pods'?
If so, wondering if should simplify to:
```
{{ toYaml .placement | omit 'name' 'pods' }}
```

Reply:
Will try. Need omit 'schedulerName' which need special treatment.